io_section.tex

```diff
@@ -0,0 +1,257 @@
+  \item Client Ratio (CR).The percentage of time a client (model) task waits for the server. Ideally it should be as close to zero as possible.
+  \item Client I/O time. The actual time (seconds) that the client task spends in I/O related activity
+\end{itemize} 
+
+
+
+\subsection{Set 1: Strong scaling}
+
+This set of tests used a C576 mesh (approx 17Km horizontal resolution). Pure MPI only (\$OMP\_NUM\_THREADS=1). 
+I/O setup used 12 XIOS servers with lustre stripe set count to be the same. This was the best performing setup from
+the last set of I/O performance tests, so the assumption is that it is a good starting point.
+The basic run is for 120 timesteps, with diagnostic\_frequency set to 20 timesteps. This makes for 6 writes over the course of the run.
+
+\begin{table}[ht!]
+\scriptsize
+  \begin{center}
+    \caption{Strong Scaling: Runtime}
+    \label{tab:table1}
+     \begin{tabular}{|c|c|c|c|c|c|}
+      \textbf{Nodes} & \textbf{Max CR~(\%)} & \textbf{Max Client IO~(s)} & \textbf{Runtime~(s)} & \textbf{Runtime no IO~(s)} & \textbf{\% incr IO}\\
+      \hline
+      24 & 0.002 & 12.76 & 10115 & 9832 & +3\\
+      54 & 0.002 & 13.30 & 4950 & 4709 & +5\\
+      96 & 0.006 & 13.36 & 2795 & 2810 & -0.5\\
+      216 & - & - & - & - & - \\
+      384 & - & - & - & - & - \\
+    \end{tabular}
+  \end{center}
+\end{table}
+
+\begin{table}[ht!]
+\scriptsize
+  \begin{center}
+    \caption{Strong Scaling: Memory Usage *}
+    \label{tab:table2}
+     \begin{tabular}{|c|c|c|c|}
+      \textbf{Nodes} & \textbf{Memory Usage Gb} & \textbf{Memory Usage Gb (No IO)} & \textbf{\% incr IO}\\
+      \hline
+      24 & 841.3 & 764.3 & +10 \\
+      54 & 1020.2  & 936.8 & +8 \\
+      96 & ? & 1126.4 & ? \\
+      216 & - & - & -  \\
+      384 & -  & -  & -  \\
+    \end{tabular}
+  \end{center}
+\end{table}
+
+\scriptsize \textbf{* Note some job output does not include the PBS epilogue and so memory usage is not available (indicated by ?)}
+\newline
+
+\normalsize
+The results are very similar to those done in the 2016/2017 tests (which were done with a C288 mesh). For the 24, 54 and 96 node jobs we see
+evidence that scaling is preserved from the decreasing run times. The Client Ratio (CR) is very close to zero in all cases, 
+showing that model client tasks are hardly waiting at all for I/O requests to be processed and that I/O is asynchronous.
+The \% runtime penalty for I/O is low and well within the assumed machine variability. The \% memory usage penalty for I/O is approx 8-10\%. 
+
+216 and 384 node jobs crashed out within the first few timesteps with XIOS-related errors even though no I/O was
+actually happening at those timesteps. I am assuming that further tuning is needed for these higher node-count jobs.
+See the Conclusions section for suggested avenues to explore.
+
+\subsection{Set 2: In-situ Post-processing}
+
+In this set the focus is on regridding and simple time-meaning, with usage examples taken from latest Aquaplanet branch.
```

Comment:
I'm a little confused, Are the examples actually taken from the Aquaplanet branch (i.e. are they hard coded into the share branch) or are they from a suite or are they one and the same thing?